sympy/calculus/hamilton.py

```diff
@@ -0,0 +1,75 @@
+"""
+This module implements a method to find
+Hamilton's Equations of motion for given Hamiltonian.
+"""
+from sympy import Function, sympify, diff, Eq, S, Symbol, Derivative
+from sympy.core.compatibility import (iterable, range)
+
+
+def hamilton_equations(H, coords, momenta, time):
+    r"""
+    Find the Hamilton's equations [1]_ for a given Hamiltonian.
```

Comment:
I don't think you need the citation reference here

Reply:
Yeah, didn't see that, got rid of it, thanks.